### ISSUE TITLE
[BUG] <title>Qwen2-7b-instruct使用SFT-FT，loss变为0，如何解决？

### ISSUE THREAD

**Issue by ming-shy** [QwenLM/Qwen#1299]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

我的训练参数如下：
export CUDA_VISIBLE_DEVICES="1,6,7" && DS_SKIP_CUDA_CHECK=1 OMP_NUM_THREADS=8 torchrun --standalone --nproc_per_node=gpu train_qwen.py
--model_name_or_path $model_path
--deepspeed $deepspeed_config
--data_path $data_path
--bf16 True
--output_dir $output_path
--num_train_epochs 1
--max_steps 376
--per_device_train_batch_size 1
--gradient_accumulation_steps 16
--evaluation_strategy "no"
--save_strategy "steps"
--save_steps 47
--save_total_limit 1
--learning_rate 1e-5
--weight_decay 0.1
--adam_beta2 0.95
--warmup_ratio 0.01
--lr_scheduler_type "cosine"
--logging_steps 8
--model_max_length 8192
--gradient_checkpointing True
--lazy_preprocess False
--iterable True
--tf32 0

如下是我的运行日志，第二步loss就变为0了。不知道问题是什么
{'loss': 1.7328, 'grad_norm': 1.9622441768332994e-05, 'learning_rate': 9.919354838709679e-06, 'epoch': 0.02}
{'loss': 0.0, 'grad_norm': 3.913150697629098e-06, 'learning_rate': 9.704301075268819e-06, 'epoch': 0.04}
4%|████▉ | 16/376 [17:36<6:36:00, 66.00s/it]

### 期望行为 | Expected Behavior

能够正常训练

### 复现方法 | Steps To Reproduce

使用的fscaht微调代码，chaglm可正常训练，Qwen系列模型无法正常训练，特殊token使用的是qwen系列的<|im_start|>、<|im_end|>

### 运行环境 | Environment

```Markdown
- OS: ubuntu20.04
- Python:3.10
- Transformers: 4.34
- PyTorch: 2.3
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):  12.2
```

### 备注 | Anything else?

使用Qwen2微调也一样的现象。

**Comment by jesuswa** [comment]
你这个问题是怎么解决的
